handout/features/destructuring.md

```diff
@@ -52,8 +52,8 @@ sayName(jane) // -> Hello Jane N/A Doe
 sayName(john) // -> Helo John Smith Doe
 ```
 
-There are _many_ more sophisticated things that can be done with destructuring,
-and the [mdn][mdnDest] has some great examples, including nested Object
-destructuring, and dynamic destructing during for-ins iterators.
+There are many more sophisticated things that can be done with destructuring,
+and the [mdn][mdnDest] has some great examples, including nested object
+destructuring and dynamic destructing during for-ins iterators.
```

Comment:
Yeah this does make sense I think to a JS programmer.

Perhaps:

"... dynamic destructuring with `for ... in` operators".